src/pages/index.js

```diff
@@ -0,0 +1,3 @@
+import { Home } from "./home/home";
```

Comment:
I'd recommend to add `index.js` file with this export to avoid "./home/home" and do just "./home"

Reply:
Solved